modules/client/src/client-common/interfaces/plugin.ts

```diff
@@ -243,3 +242,42 @@ export type SubgraphVotingSettings = {
   supportThreshold: string;
   votingMode: VotingMode;
 };
+
+export enum PrepareInstallationStep {
+  PREPARING = "preparing",
+  DONE = "done",
+}
+
+export type PrepareInstallationStepValue =
+  | { key: PrepareInstallationStep.PREPARING; txHash: string }
+  | {
+    key: PrepareInstallationStep.DONE;
+  } & ApplyInstallationParams;
+
+export type ApplyInstallationParamsBase = {
```

Comment:
why isn't the helpers hash included in here?

Reply:
Because for the `ApplyInstallationParams` I need the helpers as an array of strings, like they are returned by the prepareInstallation, when the ApplyInstallationAction is called the helpers get hashed, so in the decoded ones ther is no way of sharing the same prameter so I have two, one for each case `helpers: []string` and `helpersHash`.

The other thing we could do is use always the helpers hash and expose a function to hash to the user then we woudn't have differences between the encoded and the decoded parameters